.vscode/settings.json

```diff
@@ -21,11 +21,14 @@
     "docs/**": true
   },
   "files.eol": "\n",
-  // Disable default VSCode formatting for JavaScript
+  // Disable default VSCode formatting for JavaScript	   // Disable default VSCode formatting for JavaScript
```

Comment:
Should not add unrelated codes in feature PR

Reply:
done